doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+
+.. image:: images/ID.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: ID
+    :height: 10cm
+
+we set the inlet velocity to 2.5 m/s, and we do not impose the background velocity on the bottom of the cylinder as in the previous spouted bed example. The value of beta on the outlet boundary was set to 100, which is relatively high, to stabilize the simulation.
+
+.. code-block:: text
+
+  subsection boundary conditions
+    set time dependent = false
+    set number         = 5
+
+
+    subsection bc 0 #outlet
+      set id   = 3
+      set type = outlet
+      set beta = 100
+    end
+
+    subsection bc 1 #inlet
+      set id   = 2
+      set type = function
+      subsection u
+        set Function expression = 0
+      end
+      subsection v
+        set Function expression = 0
+      end
+      subsection w
+        set Function expression = 2.5
+      end
+    end
+
+    subsection bc 2 #bed_wall
+      set id = 6
+      set type = slip
+    end
+
+    subsection bc 3 #channel_wall
+      set id = 5
+      set type = slip
+    end
+    
+    subsection bc 4   #bed_wall_bottom
+      set id   = 4
+      set type = function
+      subsection u
+        set Function expression = 0
+      end
+      subsection v
+        set Function expression = 0
+      end
+      subsection w
+        set Function expression = 0.5
+      end
+    end
+  end
+
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_.
+
+------------------------------
+Running the CFD-DEM Simulation
+------------------------------
+
+The simulation is run using the ``lethe-fluid-particles`` application. Assuming that the ``lethe-fluid-particles`` executable is within your path, the simulation can be launched as per the following command:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-fluid-particles gas-solid-spouted-bed.prm
+
+---------
+Results
+---------
+
+We briefly discuss the results that can be obtained from this example here.
+
+Total Pressure Drop
+~~~~~~~~~~~~~~~~~~~
+
+We show the data about pressure drop, which is a very important value for the fluidization phenomena.
+
+This graph illustrates the variation of pressure drop from 1s to 5s. We can see the pressure oscillation which is caused by the bubbly state.
+
+.. image:: images/pressure_drop.png
+    :alt: Pressure drop as a function of time
+    :align: center
+    :name: press_t
+
+The graph below shows pressure drop as a function of height, depending on time. Here we see the oscillation of the pressure drop as well.
+
+.. image:: images/pressure_height.gif
+    :alt: Pressure drop as a function of height
+    :align: center
+    :name: press_h
```

Comment:
I would remove this and replace it with a static graph. Gifs are heavy files to store on the repository and this does not add much information to be honest. It would be better to have a pressure drop vs time graph than this one.

Reply:
Well you already have a pressure drop vs time, so that's already great. I would just remove this figure then.